src/target/stm32f1.c

```diff
@@ -418,11 +428,15 @@ static bool at32f40_detect(target_s *target, const uint16_t part_id)
 	target->target_options |= STM32F1_TOPT_32BIT_WRITES;
 	target->mass_erase = stm32f1_mass_erase;
 
+	// AT32F403A/F407 have 48 bytes of User System Data
```

Comment:
This is a useful tidbit to know but we don't see how it relates to the line after it or how it impacts things. Please flesh this out as a comment perhaps around line 424 as part facts and how they reflect onto the implementation.

Reply:
Ah, okie - in that case perhaps prefix this with `NB: ` or so to indicate that you're delivering some extra information?

Reply:
Expanded oneline comment to carry more meaning.